sources/packages/backend/apps/queue-consumers/src/main.ts

```diff
@@ -29,10 +30,20 @@ import { SystemUsersService } from "@sims/services";
   // Create bull board UI dashboard for queue management.
   const serverAdapter = new ExpressAdapter();
   serverAdapter.setBasePath("/admin/queues");
-  const bullBoardQueues = queues.map((queue) => {
-    return new BullAdapter(app.get<Queue>(`BullQueue_${queue.name}`), {
-      readOnlyMode: queue.dashboardReadonly,
-    });
+  const bullBoardQueues: BullAdapter[] = [];
+  queues.forEach((queue) => {
+    if (
+      !queue.isActive &&
+      schedulerQueueNames.includes(queue.name as QueueNames)
```

Comment:
The `schedulerQueueNames` array  does not seem required since we can get the same result from 
```ts
Object.values(QueueNames)
```
Either way, is there a reason why we would not be relying only on the DB list?

Reply:
I wasn't going for this list until I saw one of the AC: "Non-schedulers queues will not be affected by the is_active flag for now." I then thought that we should make sure the non-scheduler queue is still running even though the `is_active` flag false. If we are going to rely on the DB list, I can remove this condition.